### ISSUE TITLE
508-defect-4: Conditional list section on Resources FE template

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-cms#13137]
## Details

On Resources pages, under  the "Need more help?" section, there is the ability for the editor to add various contact options.  These options will display in a list:
![image](https://user-images.githubusercontent.com/106678594/229209880-471c4e69-a67e-4b5a-a8c8-9131d93dc351.png)

However, in some cases the editor my only add one item to this section, most likely in this case they would choose to add the default contct number and nothing else.  In these cases, this default option ("MyVA411 main information line: 800-698-2411) will still display on the FE end in a list

This section of the template should be changed to be conditional, so that if there is only one item it is displayed in a paragraph but if there are multiple items a list will display.

## Acceptance Criteria

- [ ] Check for other places (besides R&S Detail template below) where a need-more-help section exists, as this will drive the decision of whether to componentize or just one-off code
- [ ] Need more help's content is a `<p>` if there is only 1 item
- [ ] Need more help's content is a `<ul>` with `<li>` option if there is more than one item

## Steps to Recreate

1. Enter `https://www.va.gov/resources/how-to-get-free-language-assistance-from-va/` in browser
2. Inspect the content below "need more help"
3. Confirm it is a `ul`, not a `p`

## Proposed Solution (if known)

Remove the `ul` `li` and place "MyVA411 main information line: 800-698-2411" within a `p` element.
For example:

```html
<p class="vads-u-margin-top--1">
  <strong>MyVA411 main information line: </strong>
  <a href="tel:1-800-698-2411" rel="noreferrer noopener" id="8e32ea4163003e630dacb9e1a9f31c01" lang="en">
    800-698-2411
  </a>
</p>
```

## Screenshots or Trace Logs

<img width="1596" alt="Screen Shot 2021-07-20 at 11 08 50 AM" src="https://user-images.githubusercontent.com/14154792/126349428-2e2399a7-7d41-43d9-88df-bdb4a1f2227c.png">

**Comment by wesrowe** [comment]
@randimays @chriskim2311, this ticket could use some pre-refinement from you: grok the recommended solution, make sure you agree with it, and estimate. Any questions can be directed to @laflannery.

**Comment by randimays** [comment]
content-build has been building really slowly for me locally and erroring out. This has been a barrier to me making much progress. This will carry over to the next sprint.

**Comment by randimays** [comment]
Verified in prod. Closing!
